relm4-macros/relm4-macros-internal/src/widgets/parse/view_widgets.rs

```diff
@@ -13,6 +13,12 @@ impl Parse for ViewWidgets {
 
         while input.peek(Token![,]) {
             let _colon: Token![,] = input.parse()?;
+
+            // Allow ending with a comma after the last widget
```

Comment:
This should really be it's own commit and probably a separate PR which could be merged immediately. Trailing comma support on the widgets is one of the rough edges of relm4 that I've encountered too often.